### ISSUE TITLE
Cope with Cabal API Changes

### ISSUE THREAD

**Issue by jimcrayne** [hdevtools/hdevtools#15]
I had to make some tweaks to compile with later cabal, see pull request #14

**Comment by ch1bo** [comment]
Hey sorry I forgot about your PR! I will try to have a look on it today. 

Even though I haven't tried your changes yet, I could not find that API change with `renderGhcOptions`?

**Comment by jimcrayne** [comment]
Hey, I made another commit, that particular API change looks to have happened at commit 6ae8733.
I noticed that commit had Cabal.cabal version 1.23.2.0 not 1.25.0.0, so I made another commit to update the pull request to match that.  

I used roughly the procedure I described on the pull request comment to test against commit 6ae8733 of the cabal repository, and it built.

If you want I can combine all this into one commit, but that means you'll have to reset any version of the fork repository you have locally. Let me know if you prefer that.
